assets/js/blocks/cart-checkout/cart-i2/frontend.js

```diff
@@ -45,30 +47,36 @@ const CartFrontend = ( props ) => {
 
 const getProps = ( el ) => {
 	return {
-		emptyCart: el.innerHTML,
-		attributes: getValidBlockAttributes( blockAttributes, el.dataset ),
+		attributes: getValidBlockAttributes(
```

Comment:
`getValidBlockAttributes` isn't needed any more that we render attributes in HTML. Last I tested with my dummy block, we can test here.